src/prefect/new_flow_engine.py

```diff
@@ -355,11 +355,32 @@ def get_hooks(self, state: State, as_async: bool = False) -> Iterable[Callable]:
         if not flow_run:
             raise ValueError("Task run is not set")
 
+        enable_cancellation_and_crashed_hooks = (
+            os.environ.get(
+                "PREFECT__ENABLE_CANCELLATION_AND_CRASHED_HOOKS", "true"
```

Comment:
perhaps
```suggestion
                "PREFECT_ENABLE_CANCELLATION_AND_CRASHED_HOOKS", "true"
```

Reply:
This needs to match what the `Runner` sets: https://github.com/PrefectHQ/prefect/blob/82a8383982f923d9cbf4b428913ef6636ef17fa9/src/prefect/runner/runner.py#L557, I'm going to leave this as is.

Reply:
ahh i see - thanks for explaining!